### PR TITLE
add ADS cover platform

### PR DESCRIPTION
**Pull request by carstenschroeder**
**Description:**
Adds new cover platform for ADS integration

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#23377

## Checklist:

- [ ] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [ ] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/ads.markdown [hunk 1]
```diff
@@ -14,13 +14,15 @@ ha_category:
...
   - Switch
+  - Cover
 ha_release: "0.60"
 ha_iot_class: Local Push
 redirect_from:
   - /components/binary_sensor.ads/
   - /components/light.ads/
   - /components/sensor.ads/
   - /components/switch.ads/
+  - /components/cover.ads/
```
Comment (klaasnicolaas) [inline source/_components/ads.markdown]: This is not necessary and can be removed

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_components/ads.markdown (hunk 1 above)
Comment (carstenschroeder) [reply source/_components/ads.markdown]: The whole redirect_from chapter?

Comment (klaasnicolaas) [reply source/_components/ads.markdown]: No only the `cover.ads`

Comment (carstenschroeder) [reply source/_components/ads.markdown]: done

**Review by MartinHjelmare — APPROVED** [review]

**Review by klaasnicolaas — APPROVED** [review]

**Pull request merged**